test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperDisplayExtensionsTest.cs

```diff
@@ -355,6 +414,22 @@ public void DisplayForModel_UsesTemplateNameAndHtmlFieldName()
         private class SomeModel
         {
             public string SomeProperty { get; set; }
+
+            public Status Status { get; set; }
+        }
+
+        private class StatusResource
+        {
+            public static string Type_Fail = "failure from type";
+        }
+
+        private enum Status : byte
+        {
+            [Display(Name = "Resx_Pass")]
+            Pass,
+            [Display(Name = "Type_Fail", ResourceType = typeof(StatusResource))]
```

Comment:
Let's do something less confusing like `Created`, `Started`, `Faulted`, and `DoneDoneDone`.